Trading/Exchange/okex_websocket_feed/okex_websocket.py

```diff
@@ -22,13 +22,10 @@ class OKExCryptofeedWebsocketConnector(exchanges.CryptofeedWebsocketConnector):
     REQUIRED_ACTIVATED_TENTACLES = []
     EXCHANGE_FEEDS = {
         Feeds.TRADES: cryptofeed_constants.TRADES,
-        Feeds.KLINE: Feeds.UNSUPPORTED.value,
         Feeds.TICKER: cryptofeed_constants.TICKER,
-        Feeds.CANDLE: Feeds.UNSUPPORTED.value,
```

Comment:
Feeds.CANDLE not `Feeds.UNSUPPORTED.value` is not a problem ? considering it has many static usages

Reply:
No, I with the new integration it works properly